src/scancode/cli.py

```diff
@@ -356,6 +379,12 @@ def print_options(ctx, param, value):
     help='Show the list of available ScanCode plugins and exit.',
     help_group=DOC_GROUP, cls=CommandLineOption)
 
+@click.option('--list-packages',
```

Comment:
This should go in the packagedcode module and plugin instead

Reply:
@pombredanne  Got it, thanks. I will update according to this idea.